codespell_lib/data/dictionary.txt

```diff
@@ -14954,6 +14960,11 @@ inexpirienced->inexperienced
 infact->in fact
 infalability->infallibility
 infallable->infallible
+infaltable->inflatable, infallible,
+infalte->inflate
+infalted->inflated
+infaltes->inflates
```

Comment:
Do you think it's also worth adding `infants` here, or is that too different to the original typo?

Reply:
It feels rather a stretch to me, it's `lte`->`n` and all of those letters are at least two away from n on QWERTY, so it seems rather unlikely to me.